UnrealPlugin/ThirdPersonMPCloudDeploy.md

```diff
@@ -15,28 +15,81 @@ This guide explains how to deploy your game to PlayFab Multiplayer Server (MPS)
 * [Completed Unreal Project](ThirdPersonMPSetup.md) with [PlayFab Unreal GSDK](ThirdPersonMPGSDKSetup.md) installed and configured
 * [Release Server](ThirdPersonMPBuild.md) configuration of your project built from Visual Studio or Development Editor
 * [Client](ThirdPersonMPBuild.md) configuration of your project built from Visual Studio or Development Editor
+* MPS is [enabled for your title](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/enable-playfab-multiplayer-servers), and billing is set up
 
 ## Instructions
 
 ### Packaging the game server
 
-Make sure that you have just built your project in development editor in Visual Studio *before* doing this next step, because sometimes building fails when opening the project directly through the .uproject file.
-
-Right click on the .uproject file in your file editor and select "Generate Visual Studio Project Files".
-
-Then open the .uproject file.
-
-In the top left of the Unreal editor menu, click on File -> Package -> Target Configuration -> [game name]Server and then go to File -> Package -> Windowsx64.
-You can now use this packaged version of your game server to [test with LocalMultiplayerAgent](https://docs.microsoft.com/gaming/playfab/features/multiplayer/servers/locally-debugging-game-servers-and-integration-with-playfab)
-or to use it directly with PlayFab by [creating a build](https://developer.playfab.com/) and then using [PlayFab MpsAllocatorSample](https://github.com/PlayFab/MpsSamples/blob/master/MpsAllocatorSample/README.md).
+* [OPTIONAL] This section can be done to refresh the files which Unreal builds into your project. This is useful if you've recently created or deleted files.
+	* If you have not recently built your project in "development editor" configuration, do so now
+	* Close Visual Studio
+	* Navigate to the .uproject file for your project
+	* Right click on the .uproject file in your file editor and select "Generate Visual Studio Project Files"	
+	* Re-open the {ProjectName}.sln file in Visual Studio
+* Choose "development editor" configuration
+* Run your project
+* Observe the Unreal Editor, which should have loaded your project
+* In the top left of the Unreal editor menu, click on File -> Package -> Target Configuration -> {ProjectName}Server
+* In the top left of the Unreal editor menu, click on File -> Package -> SOMETHING SOMETHING -> Shipping
+* File -> Package -> Windowsx64
```

Comment:
__File __>__Package __>__ Windowsx64__

Formatting.

Note that I added extra space so it shows up here.